module/scratchpad.lua

```diff
@@ -50,11 +50,45 @@ function Scratchpad:turn_on()
         if self.reapply then self:apply(c) end
         -- c.sticky was set to false in turn_off so it has to be reapplied anyway
         c.sticky = self.sticky
+        local new_y = c.y
+        local new_x = c.x
+
+        -- Get the tweens
+        local anim_x = self.awestore.x
```

Comment:
This is accessing the awestore field without checking if it was set. You would want to add `info.awestore = info.awestore or {}` in the `new` function to let the user not set the awestore field at all.